source/configuration/sharding.txt

```diff
@@ -0,0 +1,161 @@
+sharded with the specified shard key. When {+odm+} reloads models, it provides
+the shard key with the ``_id`` field to the ``find`` command to improve query
+performance. If the collection is not sharded with the specified shard key,
+queries might not return the expected results.
+
+You can declare shard keys by using either the standard MongoDB
+syntax, or by using a shorthand syntax.
+
+Standard Syntax
+~~~~~~~~~~~~~~~
+
+The standard syntax follows the format
+of the ``mongosh`` :manual:`shardCollection()
+</reference/method/sh.shardCollection>` method, and allows you to specify the
+following types of shard keys:
+
+- Ranged keys
+- Hashed keys
+- Compound keys
+
+The standard syntax also allows you to specify collection and sharding options.
+
+The following example creates a shard key on the ``ssn`` field in each of the
+preceding formats:
+
+.. literalinclude:: /includes/configuration/sharding.rb
+   :language: ruby
+   :start-after: # start-shard-key-formats
+   :end-before: # end-shard-key-formats
+
+Shorthand Syntax
+~~~~~~~~~~~~~~~~
+
+The shorthand syntax allows you to declare a shard key by specifying only the
+field name. This syntax supports only ranged and compound shard keys, and does not allow you
+to specify collection or sharding options.
+
+The following example creates a ranged and a compound shard key:
+
+.. literalinclude:: /includes/configuration/sharding.rb
+   :language: ruby
+   :start-after: # start-shard-key-shorthand
+   :end-before: # end-shard-key-shorthand
+
+Specify Associated and Embedded Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify a shard key on a ``belongs_to`` association in place of a field
+name. When doing so, {+odm+} creates the shard key on the primary key of the
+associated collection.
+
+The following example creates a shard key on the ``belongs_to`` association in a
+``Person`` model. Because the associated ``country`` collection has a primary
+key called ``country_id``, {+odm+} shards by that field:
+
+.. literalinclude:: /includes/configuration/sharding.rb
+   :language: ruby
+   :start-after: # start-shard-key-association
+   :end-before: # end-shard-key-association
+
+You can specify a shard key on an embedded document by using dot notation to
+delimit the field names. The following example creates a shard key on the
+``address.city`` field:
+
+.. literalinclude:: /includes/configuration/sharding.rb
+   :language: ruby
+   :start-after: # start-shard-key-embedded
+   :end-before: # end-shard-key-embedded
+
+.. note::
+
+   Because the ``.`` character is used to delimit embedded fields, {+odm+} does
+   not support creating shard keys on fields with names that contain the ``.``
+   character.
+
+Sharding Management Rake Tasks
+------------------------------
+
+You can shard collections in your database according the the shard keys defined
+in your {+odm+} models by running the ``db:mongoid:shard_collections`` Rake
```

Comment:
I: all the [docs](https://www.mongodb.com/docs/mongoid/current/reference/indexes/) i could find use lowercase-r 'rake'

```suggestion
in your {+odm+} models by running the ``db:mongoid:shard_collections`` rake
```

applies throughout